### ISSUE TITLE
please delete the `deform2` branch

### ISSUE THREAD

**Issue by tisdall** [Pylons/deform#279]
The current `master` branch is the lastest version of deform which is 2.0.  However, there's also a dead branch called `deform2` which is currently 138 commits behind the master with nothing not already ready in the master branch.  This is a source of confusion.

In general, it'd also be nice to simply delete all the stale branches that have already been merged into the  `master` and are no longer valid, but `deform2` seems to be the largest source of confusion.

**Comment by tseaver** [comment]
I deleted all the stale branches with '| 0' showing just now.

**Comment by tisdall** [comment]
fastest.  response.  ever.  ;)

**Comment by tisdall** [comment]
ugh..  it seems that PR's related to the dead branch were also killed.  Sorry, I didn't see that there were PR's attached to that branch.  I guess those people could update their PR and resubmit.

**Comment by dairiki** [comment]
So there were three PRs submitted by myself (#184, #185, #187) which were just closed.  I'm happy to put some work into them to rebase to current master and clean out any code rot, but... these pull requests have been sitting around feeling neglected for nearly two years now.  If I do clean them up, what's the chance that it will lead somewhere?

**Comment by stevepiercy** [comment]
@dairiki as of today, here's the latest discussion on the matter, [starting from this message](https://botbot.me/freenode/pyramid/2015-09-15/?msg=49778930&page=3).

tl;dr: There is currently no active maintainer of deform, and finding a qualified maintainer is hard.

**Comment by stevepiercy** [comment]
@dairiki I am now an active maintainer of Deform on both PyPI and GitHub. Sorry it took so long.

I would like to get your PRs #185 and #187 into Deform 3.0 (#184 made it in via https://github.com/Pylons/deform/pull/280). Would you please create a PR against the `master` branch?

[Lots of work has already taken place on deformdemo](https://github.com/Pylons/deformdemo/blob/master/CHANGES.txt). In Deform I'm starting triage of open PRs now, and issues later this week. Thank you!
